packages/core/android/src/main/java/io/sentry/react/RNSentryStart.java

```diff
@@ -0,0 +1,307 @@
+package io.sentry.react;
+
+import android.app.Activity;
+import android.content.Context;
+import com.facebook.react.bridge.ReadableMap;
+import com.facebook.react.bridge.ReadableType;
+import com.facebook.react.common.JavascriptException;
+import io.sentry.ILogger;
+import io.sentry.Integration;
+import io.sentry.SentryEvent;
+import io.sentry.SentryLevel;
+import io.sentry.SentryReplayOptions;
+import io.sentry.UncaughtExceptionHandlerIntegration;
+import io.sentry.android.core.AnrIntegration;
+import io.sentry.android.core.BuildConfig;
+import io.sentry.android.core.CurrentActivityHolder;
+import io.sentry.android.core.NdkIntegration;
+import io.sentry.android.core.SentryAndroid;
+import io.sentry.android.core.SentryAndroidOptions;
+import io.sentry.protocol.SdkVersion;
+import io.sentry.protocol.SentryPackage;
+import io.sentry.react.replay.RNSentryReplayMask;
+import io.sentry.react.replay.RNSentryReplayUnmask;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.List;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public class RNSentryStart {
```

Comment:
`RNSentryStart` can be a utility class as there is no internal state.

We can make all the methods static.

Reply:
Good idea @krystofwoldrich 👍 
Updated with dcb7e2e4a96291b455fd2bc162b6f21961306388